docs/usage_guide.rst

```diff
@@ -339,6 +350,19 @@ Without a premium subscription you also can't see the amount of DAI earned per a
 
 Finally you need to have premium in order for the total amount of DAI earned in a given time period to be counted in the profit/loss statements of the tax report.
 
+Borrowing
+=============
+
+In the borrowing section you can find information on your makerdao vaults. The vaults are autodetected from your ethereum accounts and information about them is shown.
+
+As a normal non-premium user you can see all your vaults, and for each one inspect the locked collateral, collateralization, debt generated and the liquidation price.
+
+With a premium subscription you can also see additional information such as the creation time of the vault, list of historical activities, total interest owed and liquidation events.
+
+Premium users can also have makervault interest taken into account in the profit/loss report.
+
+Finally premium users can create watchers for their vaults. They can add a target collateralization ratio they would like rotki to watch for in a vault. If the collateralization ration becomess less/greater than that ratio then an alert is sent to the user's email. This watcher service runs in the rotki server so you don't even need to leave the application open.
```

Comment:
becomess => becomes

runs in => runs on

the user's => your email (since we use second person throughout)